### PR TITLE
[12.0][ADD] fieldservice_account_payment

### PR DESCRIPTION
**Pull request by murtuzasaleh**
#381

### CODE REVIEW

**Comment by max3903** [comment]
@murtuzasaleh Please review and merge https://github.com/murtuzasaleh/field-service/pull/5

**Comment by max3903** [comment]
@wolfhall Can we get a demo/test Google API key to add to the tests and fix the error here:
https://travis-ci.org/OCA/field-service/jobs/628707262#L2338

File: fieldservice_account_payment/models/fsm_order.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    @api.multi
+    def action_view_payments(self):
+        action = self.env.ref('account.action_account_payments').read()[0]
+        if self.payment_count > 1:
+            action['domain'] = [('id', 'in', self.payment_ids)]
+        elif self.payment_ids:
+            action['views'] = \
+                [(self.env.ref('account.view_account_payment_form').id,
+                  'form')]
+            action['res_id'] = self.payment_ids[0]
```
Comment (max3903) [inline fieldservice_account_payment/models/fsm_order.py]:
```suggestion
            action['res_id'] = self.payment_ids[0].id
```

**Comment by murtuzasaleh** [comment]
> @wolfhall Can we get a demo/test Google API key to add to the tests and fix the error here:
> https://travis-ci.org/OCA/field-service/jobs/628707262#L2338

@max3903 Done, Travis is green.

**Review by max3903 — APPROVED** [review]

**Review by hparfr — APPROVED** [review]

**Review by brian10048 — APPROVED** [review]

**Pull request merged**